R/fotmob_players.R

```diff
@@ -61,10 +62,10 @@ fotmob_get_match_players <- function(match_ids) {
           xt2 <- matrix(
             as.numeric(gsub("%", "", xt)),
             ncol = ncol(xt)
-          )
+          ) %>%
+            tibble::as_tibble()
```

Comment:
moved this to avoid the warning that tibble prints out about `as_tibble.matrix` the first time this is called in a session